### PR TITLE
Add configuration for ready-to-go downloaded sample - python daemon app

### PR DESCRIPTION
**Pull request by archieag**
@jmprieur please review the update. We'd need AppCreationScripts/apps.json to be in the root folder for code replacement to work seamlessly.

### CODE REVIEW

File: AppCreationScripts/apps.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                "id": "df021288-bdef-4463-88db-98f22de89214",
+                "type": "Role",
+                "x-ms-name": "User.Read.All"
+              }
+            ]
+          }
+        ],
+       "codeConfigurations": [	
+        {
+		      "settingFile": "ms-identity-python-daemon/1-Call-MsGraph-WithSecret/parameters.json",
```
Comment (jmprieur) [inline AppCreationScripts/apps.json:47]: no leading slash?

**Review by jmprieur — COMMENTED** [review]
LGTM. 
I've one question

File: AppCreationScripts/apps.json (hunk 1 above)
Comment (archieag) [reply AppCreationScripts/apps.json:47]: yes, no leading slash

**Comment by archieag** [comment]
@jmprieur Please merge.

File: AppCreationScripts/apps.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+    /*
+    This section describes the Azure AD Applications to configure, and their dependencies
+    */
+  
+    "Sample": {
+      "Title": "Acquire a token and call Microsoft Graph API from a console app using app's identity",
```
Comment (jmprieur) [inline AppCreationScripts/apps.json:7]: from a Python daemon console app?

**Review by jmprieur — APPROVED** [review]
LGTM. Thanks @archieag

File: AppCreationScripts/apps.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+    /*
```
Comment (rayluo) [inline AppCreationScripts/apps.json:2]: Perhaps Microsoft portal will remove this automatically, but ideally we should not add comments like this into a json file. JSON standard does not define/allow comment.

**Pull request merged**